spec/models/category_spec.rb

```diff
@@ -1,5 +1,70 @@
-require 'spec_helper'
+require 'rails_helper'
 
-describe Category do
-  pending "add some examples to (or delete) #{__FILE__}"
-end
+RSpec.describe Category, :type => :model do
+
+  it { should have_many(:videos) }
```

Comment:
What I meant by this - `should` is an older RSpec thing and `is_expected.to` is the successor :) newer syntax. You are using latest RSpec, I thought you might as well know about it.